logger/logger.swift

```diff
@@ -1,10 +1,188 @@
 //
 //  logger.swift
-//  logger
+//  Test
 //
-//  Created by Daniel Brain on 10/12/20.
+//  Created by Daniel Brain on 10/11/20.
 //
 
-class logger {
+import Foundation
 
+let DEFAULT_PAYLOAD: Payload = [:]
+
+class Logger {
+    private let url: URL
+    private let prefix: String?
```

Comment:
Maybe call this `identifier` instead?

Reply:
Also in PayPal we've needed something like this to identify events, so having it as `String` instead of `String?` makes more sense as I can see the use-case is necessary I suppose

Reply:
I'd prefer to keep consistent with the existing web library, for now, if possible. Making it mandatory could work; but I think I prefer to give the flexibility to the caller.

Reply:
If existing web library has a `prefix` and is optional then makes sense